src/libs/actions/Chronos.js

```diff
@@ -0,0 +1,62 @@
+import _ from 'underscore';
+import CONST from '../../CONST';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as API from '../API';
+
+/**
+ @param {String} reportID
+ @param {String} reportActionID
+ @param {String} eventID
+ * @param {Object[]} events
+ */
+const removeEvent = (reportID, reportActionID, eventID, events) => {
+    const optimisticData = [
+        {
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`,
+            value: {
+                [reportActionID]: {
+                    pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+                    originalMessage: {
+                        events: _.filter(events, event => event.id !== eventID),
```

Comment:
I think the use of `_.reject` may be a better fit here.
Extra: Do you think we can stop the iteration once an item has been rejected? We don't really need to keep iterating over the rest.

Reply:
Good call, I like reject() or omit()